### PR TITLE
Bug fix: calling ->output() with $fields parameter set

### PR DESCRIPTION
**Pull request by Fonata**
@susgo @jimeh @williamknauss I'll merge this on March 24 unless there are concerns.

As always: feedback and criticism are welcome. See commit messages for details.

### CODE REVIEW

File: src/Csv.php [hunk 1]
```diff
@@ -364,13 +369,11 @@ public function parse($input = null, $offset = null, $limit = null, $conditions
         }
 
         if (empty($input)) {
-            // todo: but why true?
-            return true;
+            return false;
```
Comment (susgo) [inline src/Csv.php]: 👍

File: src/Csv.php [hunk 2]
```diff
@@ -380,12 +383,7 @@ public function parse($input = null, $offset = null, $limit = null, $conditions
             $this->data = $this->parse_string();
         }
 
-        if ($this->data === false) {
-            return false;
-        }
-
-        return true;
-
+        return $this->data !== false;
```
Comment (susgo) [inline src/Csv.php]: 👍

**Comment by susgo** [comment]
@Fonata Die Tests für PHP 5.4 sollten wir dann rausnehmen?

**Comment by Fonata** [comment]
The tests for PHP 5.4 will be removed in PR #131. No need to do that here.

**Pull request merged**